tests/ui/conftest.py

```diff
@@ -49,16 +88,166 @@ def jwt_secret(base_url, variables):
+
+
+@pytest.fixture
+def addon(transactional_db, create_superuser, pytestconfig):
+    """
+    Creates a custom addon named 'Ui-Addon'. This addon will be a featured
+    addon and will have a featured collecton attatched to it. It will
+    belong to the user created by the 'create_superuser' fixture.
+
+    It has 1 preview, 5 reviews, and 2 authors. The second author is named
+    'ui-tester2'. It has a version number as well as a beta version.
+    """
+
+    if not pytestconfig.option.usingliveserver:
+        return
+    default_icons = [x[0] for x in icons() if x[0].startswith('icon/')]
+    addon = addon_factory(
+        status=STATUS_PUBLIC,
+        type=ADDON_EXTENSION,
+        average_daily_users=4242,
+        users=[UserProfile.objects.get(username='uitest')],
+        average_rating=4.21,
+        description=u'My Addon description',
+        file_kw={
+            'hash': 'fakehash',
+            'platform': amo.PLATFORM_ALL.id,
+            'size': 42,
+        },
+        guid=generate_addon_guid(),
+        homepage=u'https://www.example.org/',
+        icon_type=random.choice(default_icons),
+        name=u'Ui-Addon',
+        public_stats=True,
+        slug='ui-test',
+        summary=u'My Addon summary',
+        support_email=u'support@example.org',
+        support_url=u'https://support.example.org/support/ui-test-addon/',
+        tags=['some_tag', 'another_tag', 'ui-testing',
+                'selenium', 'python'],
+        total_reviews=777,
+        weekly_downloads=2147483647,
+        developer_comments='This is a testing addon, used within pytest.',
+        is_experimental=True,
+    )
+    Preview.objects.create(addon=addon, position=1)
+    Review.objects.create(addon=addon, rating=5, user=user_factory())
+    Review.objects.create(addon=addon, rating=3, user=user_factory())
+    Review.objects.create(addon=addon, rating=2, user=user_factory())
+    Review.objects.create(addon=addon, rating=1, user=user_factory())
+    addon.reload()
+    AddonUser.objects.create(user=user_factory(username='ui-tester2'),
+                             addon=addon, listed=True)
+    version_factory(addon=addon, file_kw={'status': amo.STATUS_BETA},
+                    version='1.1beta')
+    addon.save()
+    generate_collection(addon, app=FIREFOX)
+
+    print('Created addon {0} for testing successfully'.format(addon.name))
+    return addon
+
+
+@pytest.fixture
+def themes(transactional_db, create_superuser, pytestconfig):
+    """
+    Creates exactly 6 themes that will be not featured. These belong to the
+    user created by the 'create_superuser' fixture. It will also create 6
+    themes that are featured with random authors.
+    """
+
+    if not pytestconfig.option.usingliveserver:
+        return
+    owner = UserProfile.objects.get(username='uitest')
+    generate_themes(6, owner)
+    for _ in range(6):
+        addon = addon_factory(
+            status=STATUS_PUBLIC,
+            type=ADDON_PERSONA,)
```

Comment:
comma at the end not needed here. Also, is ``addon = addon_factory(status=STATUS_PUBLIC, type=ADDON_PERSONA)`` really too long?